mastercraft.cmd

```diff
@@ -750,7 +762,7 @@ calc.material:
           action var sigil %sigil|$1 when ^\s+\(\d\)\s+(?:primary|secondary) sigil \((\S+)\)
           pause .5
           if !matchre("$righthand", "book") then gosub GET my %discipline book
-          send read my book
+          PUT read my book

```

Comment:
changed from send to PUT. I was going to be changing to gosub put, to handle for lag etc, but there is more work required. Will come in a later update.